### PR TITLE
Add TrueGalaxyCash (TGCH) currency

### PR DESCRIPTION
**Pull request by galaxycash**
SPECIFICATIONS
Premine:    0.05%, 20,000 TGCH
Ticker:    TGCH
Type:    PoW/PoS Hybrid with Masternodes
Algorithm:    X12
Difficulty Retarget:    Every Block (DGWv3)
Block Time:    60 Seconds
Block Size:    6 mb
PoW Block Reward:    0,5 TGCH
PoW End Block: 20,000 block
PoS Block Reward:    DPOS
Masternode Price:    1000 TGCH
Masternode PoW Reward:    85%
Masternode PoS Reward:    80%
Min PoS Age:    4 hours
Max PoS Age:    No Max
Coinbase Maturity:    32 Blocks
Minimum Confirmations:    10 Blocks
Stake Confirmations:    32 Blocks
Max Coin Supply:    35000000 TGCH

Links
Web: https://galaxycash.info
Explorer: https://explorer.galaxycash.info
Supply access API link: https://explorer.galaxycash.info/ext/getmoneysupply
Logo: https://galaxycash.info/img/truegalaxycash.png
Bitcoin talk thread: https://bitcointalk.org/index.php?topic=4920919
Twitter: https://twitter.com/cash_galaxy
Telegram channel: https://t.me/galaxycash
Github: https://github.com/galaxycash/truegalaxycash
Wallets: https://github.com/galaxycash/truegalaxycash/releases/latest
Coinlib: https://coinlib.io/coin/TGCH/TrueGalaxyCash
Neironix: https://neironix.io/ru/cryptocurrency/galaxycash
Blockfolio: https://blockfolio.com

### CODE REVIEW

File: app/renderer/assets/custom-cryptocurrency-icons/tgch.svg [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (sindresorhus) [inline app/renderer/assets/custom-cryptocurrency-icons/tgch.svg]: We don't accept icons here. The current icons here are just for our internal coins. Open an issue on https://github.com/atomiclabs/cryptocurrency-icons and we'll have a proper icon made for you.

File: app/marketmaker/supported-currencies.js [hunk 2]
```diff
@@ -2323,6 +2323,20 @@ const supportedCurrencies = [
...
+	{
+		coin: 'TGCH',
+		rpcport: 5604,
+		pubtype: 38,
+		p2shtype: 99,
+		wiftype: 89,
+		txfee: 100,
+		electrumServers: [
+			{
+				host: '195.133.145.160',
```
Comment (sindresorhus) [inline app/marketmaker/supported-currencies.js]: `host` should be a domain. I'm aware we have some IPs as `host` included already, but for new additions, we prefer domains. There should also be at least two servers, for redundancy, preferably three.

File: app/marketmaker/supported-currencies.js [hunk 3]
```diff
@@ -2323,6 +2323,20 @@ const supportedCurrencies = [
 		etomic: '0xE41d2489571d322189246DaFA5ebDe1F4699F498',
 		rpcport: 80,
 	},
+	{
```
Comment (sindresorhus) [inline app/marketmaker/supported-currencies.js]: The list is alphabetically sorted. Please place the entry accordingly.

**Comment by galaxycash** [comment]
We not have domains for electrum servers. I add additional server and icon in nearest time.

**Comment by sindresorhus** [comment]
You can just put them on subdomains of galaxycash.info, like how it's done for many of the other currencies. For example, `electrum1.galaxycash.info`, `electrum2.galaxycash.info`, `electrum3.galaxycash.info`.

**Comment by galaxycash** [comment]
Ready
Electrum servers assigned to electrum1.galaxycash.info, electrum2.galaxycash.info

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**